src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -710,6 +711,58 @@ int SketchSelection::setUp()
 
 }// namespace SketcherGui
 
+namespace{
+enum CONSTRAINT_DISTANCE_QUALIFIER{
```

Comment:
The C++ Core Guidelines recommend using `enum class` over `enum` in new code: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class

The also discourage all-caps names for things that aren't macros:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class

Reply:
Thank you @chennes for the advices.